extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzSchedulerImpl.java

```diff
@@ -757,6 +766,14 @@ private Optional<TriggerBuilder<?>> createTrigger(String identity, Scheduled sch
         return Optional.of(triggerBuilder);
     }
 
+    private static boolean isOTelTracingEnabled() {
+        return ConfigProvider.getConfig().getValue("quarkus.otel.enabled", Boolean.class);
+    }
+
+    private static boolean isOTelExtensionAvailable() {
+        return CDI.current().select(io.opentelemetry.api.OpenTelemetry.class).isResolvable();
```

Comment:
I would try to avoid using `CDI.current().select(OpenTelemetry.class)` at all. In this particular case, I would just add the `Instance<OpenTelemetry>` injection point in the SimpleScheduler and use it here.